src/applications/claims-status/utils/helpers.js

```diff
@@ -1,11 +1,11 @@
 import merge from 'lodash/merge';
-import { format, isValid, parseISO } from 'date-fns';
 // import * as Sentry from '@sentry/browser';
 
-import environment from 'platform/utilities/environment';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 // import localStorage from 'platform/utilities/storage/localStorage';
 import { apiRequest } from 'platform/utilities/api';
 // import { fetchAndUpdateSessionExpiration as fetch } from 'platform/utilities/api';
+import { format, isValid, parseISO } from 'date-fns';
```

Comment:
Can you move these imports back up to the top?. Other than that things looks good